sklbench/datasets/__init__.py

```diff
@@ -47,9 +48,12 @@ def load_data(bench_case: BenchCase) -> Tuple[Dict, Dict]:
     dataset = get_bench_case_value(bench_case, "data:dataset")
     if dataset is not None:
         dataset_params = get_bench_case_value(bench_case, "data:dataset_kwargs", dict())
-        return dataset_loading_functions[dataset](
-            **common_kwargs, preproc_kwargs=preproc_kwargs, dataset_params=dataset_params
-        )
+        if os.environ.get("DATASETSROOT") and "train_file" in dataset_params.keys():

```

Comment:
Btw is this `DATASETSROOT` env variable usage documented?

Reply:
Good point, I will add a usage guide with details on config creation, and things like this